src/webserver.cpp

```diff
@@ -31,6 +31,7 @@
 #include "globals.h"
 #include "webserver.h"
 #include "systemcontainer.h"
+#include "improvserial.h"

```

Comment:
We have to include this to make sure `ENABLE_IMPROV_LOGGING` is available to us.